ranger/config/commands.py

```diff
@@ -1212,6 +1212,100 @@ def execute(self):
             fm.notify("files have not been retagged")
 
 
+class bulk(Command):
+    """:bulk <attribute>
+
+    This command opens a list of with the attribute <attribute> for each of the
+    selected files in an external editor.
+    After you edit and save the file, it will generate a shell script
+    which changes the attributes in bulk according to the changes you did in the
+    file.
+
+    This shell script is opened in an editor for you to review.
+    After you close it, it will be executed.
+    """
+
+
+    class id3art(object):
```

Comment:
Let's use full names everywhere.

Reply:
I actually didn't intend to add these `id3art` and `id3tit` bulk commands to ranger. I think they are very particular to my use case, and probably wouldn't see much use. I only added them in this PR to illustrate the usage of the bulk command. In any case, I have changed their names.